assets/js/components/ClusterDetails/ClusterNodeLink.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { EOS_WARNING_OUTLINED } from 'eos-icons-react';
+
+import Tooltip from '@components/Tooltip';
+import HostLink from '@components/HostLink';
+
+function ClusterNodeLink({ name, hostId }) {
```

Comment:
I would use the same strategy as `HostLink` and pass the name as `children`,
so you can use `<ClusterNodeLink hostId={id}>name</ClusterNodeLink>`